e2e/tests/hmctsSuperUserAdministersCase_test.js

```diff
@@ -12,6 +12,15 @@ BeforeSuite(async I => caseId = await I.submitNewCaseWithData(orderCaseData));
 
 Before(async I => await I.navigateToCaseDetailsAs(config.hmctsSuperUser, caseId));
 
+Scenario('HMCTS super user updates FamilyMan reference number', async (I, caseViewPage, loginPage, enterFamilyManCaseNumberEventPage) => {
+  I.seeFamilyManNumber('mockCaseID');
+  await caseViewPage.goToNewActions(config.administrationActions.addFamilyManCaseNumber);
+  enterFamilyManCaseNumberEventPage.enterCaseID('newMockCaseID');
```

Comment:
now it is not yours, but this function should be enterFamilyManCaseNumber not enterCaseId